Bad instrumentation in Math-2

Command for reproducing:
```sh
java -jar main/target/collector-sahab-0.2.1-SNAPSHOT-jar-with-dependencies.jar -p /home/aman/experiments/drr-execdiff/Math-2 -r 9c2d18b38dd7df29612bb9888c59d8cf262a7977 -l e5d67a8 -c src/main/java/org/apache/commons/math3/distribution/AbstractIntegerDistribution.java --slug ASSERT-KTH/collector-sahab-experiments --execution-depth 1 --selected-tests org.apache.commons.math3.distribution.HypergeometricDistributionTest::testMath1021 --output-path output.html
```

<details>
<summary>Stacktrace</summary>

```sh
Running org.apache.commons.math3.distribution.HypergeometricDistributionTest
rtf.org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 60: Argument 1: expected I, but found Lorg/apache/commons/math3/exception/OutOfRangeException;
        at rtf.org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:269)
        at rtf.org.objectweb.asm.util.CheckClassAdapter.verify(CheckClassAdapter.java:1074)
        at rtf.org.objectweb.asm.util.CheckClassAdapter.verify(CheckClassAdapter.java:1032)
        at io.github.chains_project.collector.CollectorAgent.getBytes(CollectorAgent.java:179)
        at io.github.chains_project.collector.CollectorAgent$1.transform(CollectorAgent.java:80)
        at java.instrument/java.lang.instrument.ClassFileTransformer.transform(ClassFileTransformer.java:246)
        at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:563)
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at org.apache.commons.math3.distribution.HypergeometricDistributionTest.makeDistribution(HypergeometricDistributionTest.java:42)
        at org.apache.commons.math3.distribution.IntegerDistributionAbstractTest.setUp(IntegerDistributionAbstractTest.java:105)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:27)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:254)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:149)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: rtf.org.objectweb.asm.tree.analysis.AnalyzerException: Argument 1: expected I, but found Lorg/apache/commons/math3/exception/OutOfRangeException;
        at rtf.org.objectweb.asm.tree.analysis.BasicVerifier.naryOperation(BasicVerifier.java:401)
        at rtf.org.objectweb.asm.tree.analysis.BasicVerifier.naryOperation(BasicVerifier.java:44)
        at rtf.org.objectweb.asm.tree.analysis.Frame.executeInvokeInsn(Frame.java:684)
        at rtf.org.objectweb.asm.tree.analysis.Frame.execute(Frame.java:609)
        at rtf.org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:165)
        ... 51 more
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.187 sec <<< FAILURE! - in org.apache.commons.math3.distribution.HypergeometricDistributionTest
testMath1021(org.apache.commons.math3.distribution.HypergeometricDistributionTest)  Time elapsed: 0.155 sec  <<< ERROR!
java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    org/apache/commons/math3/distribution/AbstractIntegerDistribution.inverseCumulativeProbability(D)I @113: invokestatic
  Reason:
    Type 'org/apache/commons/math3/exception/OutOfRangeException' (current frame, stack[1]) is not assignable to integer
  Current Frame:
    bci: @113
    flags: { }
    locals: { 'org/apache/commons/math3/distribution/AbstractIntegerDistribution', double, double_2nd, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, top, 'java/util/List' }
    stack: { 'org/apache/commons/math3/exception/OutOfRangeException', 'org/apache/commons/math3/exception/OutOfRangeException' }
  Bytecode:
    0000000: 03bd 0040 b800 463a 3612 4710 652a 04bd
    0000010: 0040 5903 bb00 4059 1249 b200 4f27 b800
    0000020: 52b7 0055 5312 57b8 005d b800 6127 0e98
    0000030: 9b00 0927 0f97 9e00 4e12 4710 662a 04bd
    0000040: 0040 5903 bb00 4059 1249 b200 4f27 b800
    0000050: 52b7 0055 5312 57b8 005d b800 61bb 003e
    0000060: 5927 b800 5203 b800 3104 b800 31b7 0064
    0000070: 59b8 0031 1265 b200 6612 57b8 005d 1936
    0000080: b800 6abf 1247 1069 2a04 bd00 4059 03bb
    0000090: 0040 5912 49b2 004f 27b8 0052 b700 5553
    00000a0: 1257 b800 5db8 0061 2ab6 006e 3e12 4710
    00000b0: 6a2a 05bd 0040 5903 bb00 4059 1249 b200
    00000c0: 4f27 b800 52b7 0055 5359 04bb 0040 5912
    00000d0: 70b2 0066 1db8 0031 b700 5553 1257 b800
    00000e0: 5db8 0061 270e 979a 004f 1247 106b 2a05
    00000f0: bd00 4059 03bb 0040 5912 49b2 004f 27b8
    0000100: 0052 b700 5553 5904 bb00 4059 1270 b200
    0000110: 661d b800 31b7 0055 5312 57b8 005d b800
    0000120: 611d 59b8 0031 1265 b200 6612 57b8 005d
    0000130: 1936 b800 6aac 1247 106d 2a05 bd00 4059
    0000140: 03bb 0040 5912 49b2 004f 27b8 0052 b700
    0000150: 5553 5904 bb00 4059 1270 b200 661d b800
    0000160: 31b7 0055 5312 57b8 005d b800 611d 1271
    0000170: a000 9012 4710 6e2a 05bd 0040 5903 bb00
    0000180: 4059 1249 b200 4f27 b800 52b7 0055 5359
    0000190: 04bb 0040 5912 70b2 0066 1db8 0031 b700
    00001a0: 5553 1257 b800 5db8 0061 2a1d b700 7427
    00001b0: 979b 0089 1247 106f 2a05 bd00 4059 03bb
    00001c0: 0040 5912 49b2 004f 27b8 0052 b700 5553
    00001d0: 5904 bb00 4059 1270 b200 661d b800 31b7
    00001e0: 0055 5312 57b8 005d b800 611d 59b8 0031
    00001f0: 1265 b200 6612 57b8 005d 1936 b800 6aac
    0000200: 1247 1072 2a05 bd00 4059 03bb 0040 5912
    0000210: 49b2 004f 27b8 0052 b700 5553 5904 bb00
    0000220: 4059 1270 b200 661d b800 31b7 0055 5312
    0000230: 57b8 005d b800 6184 03ff 1247 1076 2a05
    0000240: bd00 4059 03bb 0040 5912 49b2 004f 27b8
    0000250: 0052 b700 5553 5904 bb00 4059 1270 b200
    0000260: 661d b800 31b7 0055 5312 57b8 005d b800
    0000270: 612a b600 7736 0412 4710 772a 06bd 0040
    0000280: 5903 bb00 4059 1249 b200 4f27 b800 52b7
    0000290: 0055 5359 04bb 0040 5912 70b2 0066 1db8
    00002a0: 0031 b700 5553 5905 bb00 4059 1279 b200
    00002b0: 6615 04b8 0031 b700 5553 1257 b800 5db8
    00002c0: 0061 270f 979a 0064 1247 1078 2a06 bd00
    00002d0: 4059 03bb 0040 5912 49b2 004f 27b8 0052
    00002e0: b700 5553 5904 bb00 4059 1270 b200 661d
    00002f0: b800 31b7 0055 5359 05bb 0040 5912 79b2
    0000300: 0066 1504 b800 31b7 0055 5312 57b8 005d
    0000310: b800 6115 0459 b800 3112 65b2 0066 1257
    0000320: b800 5d19 36b8 006a ac12 4710 7d2a 06bd
    0000330: 0040 5903 bb00 4059 1249 b200 4f27 b800
    0000340: 52b7 0055 5359 04bb 0040 5912 70b2 0066
    0000350: 1db8 0031 b700 5553 5905 bb00 4059 1279
    0000360: b200 6615 04b8 0031 b700 5553 1257 b800
    0000370: 5db8 0061 2ab6 007d 3905 1247 107e 2a07
    0000380: bd00 4059 03bb 0040 5912 49b2 004f 27b8
    0000390: 0052 b700 5553 5904 bb00 4059 1270 b200
    00003a0: 661d b800 31b7 0055 5359 05bb 0040 5912
    00003b0: 79b2 0066 1504 b800 31b7 0055 5359 06bb
    00003c0: 0040 5912 7fb2 004f 1805 b800 52b7 0055
    00003d0: 5312 57b8 005d b800 612a b600 82b8 0088
    00003e0: 3907 1247 107f 2a08 bd00 4059 03bb 0040
    00003f0: 5912 49b2 004f 27b8 0052 b700 5553 5904
    0000400: bb00 4059 1270 b200 661d b800 31b7 0055
    0000410: 5359 05bb 0040 5912 79b2 0066 1504 b800
    0000420: 31b7 0055 5359 06bb 0040 5912 7fb2 004f
    0000430: 1805 b800 52b7 0055 5359 07bb 0040 5912
    0000440: 8ab2 004f 1807 b800 52b7 0055 5312 57b8
    0000450: 005d b800 6118 05b8 008e 9a00 9a18 05b8
    0000460: 0091 9a00 9218 0712 4711 0080 2a08 bd00
    0000470: 4059 03bb 0040 5912 49b2 004f 27b8 0052
    0000480: b700 5553 5904 bb00 4059 1270 b200 661d
    0000490: b800 31b7 0055 5359 05bb 0040 5912 79b2
    00004a0: 0066 1504 b800 31b7 0055 5359 06bb 0040
    00004b0: 5912 7fb2 004f 1805 b800 52b7 0055 5359
    00004c0: 07bb 0040 5912 8ab2 004f 1807 b800 52b7
    00004d0: 0055 5312 57b8 005d b800 61b8 008e 9a00
    00004e0: 1618 07b8 0091 9a00 0e18 070e 9799 0007
    00004f0: 04a7 0004 0336 0912 4711 0081 2a10 06bd
    0000500: 0040 5903 bb00 4059 1249 b200 4f27 b800
    0000510: 52b7 0055 5359 04bb 0040 5912 70b2 0066
    0000520: 1db8 0031 b700 5553 5905 bb00 4059 1279
    0000530: b200 6615 04b8 0031 b700 5553 5906 bb00
    0000540: 4059 127f b200 4f18 05b8 0052 b700 5553
    0000550: 5907 bb00 4059 128a b200 4f18 07b8 0052
    0000560: b700 5553 5908 bb00 4059 1293 b200 9615
    0000570: 09b8 0099 b700 5553 1257 b800 5db8 0061
    0000580: 1509 9904 ea12 4711 0082 2a10 06bd 0040
    0000590: 5903 bb00 4059 1249 b200 4f27 b800 52b7
    00005a0: 0055 5359 04bb 0040 5912 70b2 0066 1db8
    00005b0: 0031 b700 5553 5905 bb00 4059 1279 b200
    00005c0: 6615 04b8 0031 b700 5553 5906 bb00 4059
    00005d0: 127f b200 4f18 05b8 0052 b700 5553 5907
    00005e0: bb00 4059 128a b200 4f18 07b8 0052 b700
    00005f0: 5553 5908 bb00 4059 1293 b200 9615 09b8
    0000600: 0099 b700 5553 1257 b800 5db8 0061 0f27
    0000610: 6727 6fb8 0088 390a 1247 1100 832a 1007
    0000620: bd00 4059 03bb 0040 5912 49b2 004f 27b8
    0000630: 0052 b700 5553 5904 bb00 4059 1270 b200
    0000640: 661d b800 31b7 0055 5359 05bb 0040 5912
    0000650: 79b2 0066 1504 b800 31b7 0055 5359 06bb
    0000660: 0040 5912 7fb2 004f 1805 b800 52b7 0055
    0000670: 5359 07bb 0040 5912 8ab2 004f 1807 b800
    0000680: 52b7 0055 5359 08bb 0040 5912 93b2 0096
    0000690: 1509 b800 99b7 0055 5359 1006 bb00 4059
    00006a0: 129b b200 4f18 0ab8 0052 b700 5553 1257
    00006b0: b800 5db8 0061 1805 180a 1807 6b67 390c
    00006c0: 1247 1100 842a 1008 bd00 4059 03bb 0040
    00006d0: 5912 49b2 004f 27b8 0052 b700 5553 5904
    00006e0: bb00 4059 1270 b200 661d b800 31b7 0055
    00006f0: 5359 05bb 0040 5912 79b2 0066 1504 b800
    0000700: 31b7 0055 5359 06bb 0040 5912 7fb2 004f
    0000710: 1805 b800 52b7 0055 5359 07bb 0040 5912
    0000720: 8ab2 004f 1807 b800 52b7 0055 5359 08bb
    0000730: 0040 5912 93b2 0096 1509 b800 99b7 0055
    0000740: 5359 1006 bb00 4059 129b b200 4f18 0ab8
    0000750: 0052 b700 5553 5910 07bb 0040 5912 9db2
    0000760: 004f 180c b800 52b7 0055 5312 57b8 005d
    0000770: b800 6118 0c1d 8797 9e00 bf12 4711 0085
    0000780: 2a10 08bd 0040 5903 bb00 4059 1249 b200
    0000790: 4f27 b800 52b7 0055 5359 04bb 0040 5912
    00007a0: 70b2 0066 1db8 0031 b700 5553 5905 bb00
    00007b0: 4059 1279 b200 6615 04b8 0031 b700 5553
    00007c0: 5906 bb00 4059 127f b200 4f18 05b8 0052
    00007d0: b700 5553 5907 bb00 4059 128a b200 4f18
    00007e0: 07b8 0052 b700 5553 5908 bb00 4059 1293
    00007f0: b200 9615 09b8 0099 b700 5553 5910 06bb
    0000800: 0040 5912 9bb2 004f 180a b800 52b7 0055
    0000810: 5359 1007 bb00 4059 129d b200 4f18 0cb8
    0000820: 0052 b700 5553 1257 b800 5db8 0061 180c
    0000830: b800 a28e 0464 3e12 4711 0087 2a10 08bd
    0000840: 0040 5903 bb00 4059 1249 b200 4f27 b800
    0000850: 52b7 0055 5359 04bb 0040 5912 70b2 0066
    0000860: 1db8 0031 b700 5553 5905 bb00 4059 1279
    0000870: b200 6615 04b8 0031 b700 5553 5906 bb00
    0000880: 4059 127f b200 4f18 05b8 0052 b700 5553
    0000890: 5907 bb00 4059 128a b200 4f18 07b8 0052
    00008a0: b700 5553 5908 bb00 4059 1293 b200 9615
    00008b0: 09b8 0099 b700 5553 5910 06bb 0040 5912
    00008c0: 9bb2 004f 180a b800 52b7 0055 5359 1007
    00008d0: bb00 4059 129d b200 4f18 0cb8 0052 b700
    00008e0: 5553 1257 b800 5db8 0061 0f18 0a6f 390a
    00008f0: 1247 1100 882a 1008 bd00 4059 03bb 0040
    0000900: 5912 49b2 004f 27b8 0052 b700 5553 5904
    0000910: bb00 4059 1270 b200 661d b800 31b7 0055
    0000920: 5359 05bb 0040 5912 79b2 0066 1504 b800
    0000930: 31b7 0055 5359 06bb 0040 5912 7fb2 004f
    0000940: 1805 b800 52b7 0055 5359 07bb 0040 5912
    0000950: 8ab2 004f 1807 b800 52b7 0055 5359 08bb
    0000960: 0040 5912 93b2 0096 1509 b800 99b7 0055
    0000970: 5359 1006 bb00 4059 129b b200 4f18 0ab8
    0000980: 0052 b700 5553 5910 07bb 0040 5912 9db2
    0000990: 004f 180c b800 52b7 0055 5312 57b8 005d
    00009a0: b800 6118 0518 0a18 076b 6339 0c12 4711
    00009b0: 0089 2a10 08bd 0040 5903 bb00 4059 1249
    00009c0: b200 4f27 b800 52b7 0055 5359 04bb 0040
    00009d0: 5912 70b2 0066 1db8 0031 b700 5553 5905
    00009e0: bb00 4059 1279 b200 6615 04b8 0031 b700
    00009f0: 5553 5906 bb00 4059 127f b200 4f18 05b8
    0000a00: 0052 b700 5553 5907 bb00 4059 128a b200
    0000a10: 4f18 07b8 0052 b700 5553 5908 bb00 4059
    0000a20: 1293 b200 9615 09b8 0099 b700 5553 5910
    0000a30: 06bb 0040 5912 9bb2 004f 180a b800 52b7
    0000a40: 0055 5359 1007 bb00 4059 129d b200 4f18
    0000a50: 0cb8 0052 b700 5553 1257 b800 5db8 0061
    0000a60: 180c 1504 8798 9c00 0684 03ff 1247 1100
    0000a70: 8e2a 1006 bd00 4059 03bb 0040 5912 49b2
    0000a80: 004f 27b8 0052 b700 5553 5904 bb00 4059
    0000a90: 1270 b200 661d b800 31b7 0055 5359 05bb
    0000aa0: 0040 5912 79b2 0066 1504 b800 31b7 0055
    0000ab0: 5359 06bb 0040 5912 7fb2 004f 1805 b800
    0000ac0: 52b7 0055 5359 07bb 0040 5912 8ab2 004f
    0000ad0: 1807 b800 52b7 0055 5359 08bb 0040 5912
    0000ae0: 93b2 0096 1509 b800 99b7 0055 5312 57b8
    0000af0: 005d b800 612a 271d 1504 b600 a659 b800
    0000b00: 3112 65b2 0066 1257 b800 5d19 36b8 006a
    0000b10: ac                                     
  Stackmap Table:
    full_frame(@57,{Object[#2],Double,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})
    same_frame_extended(@132)
    full_frame(@310,{Object[#2],Double,Integer,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})
    same_frame_extended(@512)
    same_frame(@570)
    full_frame(@809,{Object[#2],Double,Integer,Integer,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})
    full_frame(@1268,{Object[#2],Double,Integer,Integer,Double,Double,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})
    same_locals_1_stack_item_frame(@1269,Integer)
    full_frame(@2103,{Object[#2],Double,Integer,Integer,Double,Double,Integer,Double,Double,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})
    full_frame(@2668,{Object[#2],Double,Integer,Integer,Double,Double,Integer,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Top,Object[#170]},{})

        at org.apache.commons.math3.distribution.HypergeometricDistributionTest.makeDistribution(HypergeometricDistributionTest.java:42)
```

</details>

--- a/trace-collector/src/main/java/io/github/chains_project/collector/CollectorAgent.java
+++ b/trace-collector/src/main/java/io/github/chains_project/collector/CollectorAgent.java
@@ -1,6 +1,5 @@
 package io.github.chains_project.collector;
 
-import static org.objectweb.asm.Opcodes.ATHROW;
 import static org.objectweb.asm.Opcodes.IRETURN;
 import static org.objectweb.asm.Opcodes.RETURN;
 
@@ -186,7 +185,7 @@ public class CollectorAgent {
     }
 
     private static boolean isItExitInstruction(int opCode) {
-        return (opCode >= IRETURN && opCode <= RETURN) || opCode == ATHROW;
+        return (opCode >= IRETURN && opCode <= RETURN);
     }
 
     private static StackManipulation.Compound getCallToEntryLogMethod(MethodNode method) throws NoSuchMethodException {

